sdcm/cluster.py

```diff
@@ -3242,7 +3242,7 @@ def node_setup(node):
         time_elapsed = time.time() - start_time
         self.log.debug('Setup duration -> %s s', int(time_elapsed))
         if not node_list[0].scylla_version:
-            result = node_list[0].remoter.run("rpm -q scylla")
+            result = node_list[0].remoter.run("rpm -q scylla-enterprise || rpm -q scylla")
```

Comment:
Thanks for the catch. How about use scylla_pkg() to get the package name?

result = node_list[0].remoter.run("rpm -q {}".format(node.scylla_pkg()))